### ISSUE TITLE
Access Denied 

### ISSUE THREAD

**Issue by AMG123456** [taniman/profit-trailer#816]
Receiving the following: 

Error Util - Error saving state ProfitTrailerData.json  (Access is denied)

How do we get this fixed.

**Comment by cristianserban** [comment]
what operating sistem ? 
did opened the file with some text editor ? if yes which one ?

**Comment by AMG123456** [comment]
Windows 10

I deleted the previous 1.6.11 version and installed the 1.6.12 version.

When that didn't work, I started from scratch and reinstalled 1.6.11.
 Hasn't worked since

On Jan 25, 2018 05:23, "Cristian Serban" <notifications@github.com> wrote:

what operating sistem ?
did opened the file with some text editor ? if yes which one ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/taniman/profit-trailer/issues/816#issuecomment-360424428>,
or mute the thread
